# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
int32 command
int32 STOW_ARM = 0
int32 STORE_OBJECT = 1
